src/services/auth.ts

```diff
@@ -1,22 +1,28 @@
 import axios from 'axios';
+import { REACT_APP_BACKEND_URL } from './constants';
 
-const BASE_API_URL: string | undefined = process.env.REACT_APP_BACKEND_URL+'/api/';
```

Comment:
keep this, you can get process.env.REACT_APP_BACKEND_URL from src/config.ts which I had defined before.